internal/server/spanner/golden/query/get_node_edges_out_bracket_props.json

```diff
@@ -15,22 +14,20 @@
     {
       "SubjectID": "geoId/5129600",
       "Predicate": "containedInPlace",
-      "ObjectID": "geoId/51620",
-      "ObjectValue": "",
-      "ObjectBytes": null,
       "Provenance": "dc/base/BaseGeos",
-      "Name": "Franklin",
+      "Value": "geoId/51620",
+      "Bytes": null,
+      "Name": "",
       "Types": [
-        "County"
+        "Thing"
```

Comment:
This is an example of the merging issue where depending on the order nodes are written, some nodes end up with default type "Thing"